content/compose/reference/_index.md

```diff
@@ -36,7 +23,7 @@ Define and run multi-container applications with Docker.
 Options:
       --ansi string                Control when to print ANSI control characters ("never"|"always"|"auto") (default "auto")
       --compatibility              Run compose in backward compatibility mode
-      --env-file stringArray       Specify an alternate environment file.
+      --env-file stringArray       Specify an alternate environment file
```

Comment:
This period is there in the cobra help string so we probably want to update it in the code as well